grid-demo/src/app/angular/row-dom-performance/row-dom-performance.component.html

```diff
@@ -30,9 +30,11 @@
     <ng-container *clrDgHideableColumn>{{ column.displayName }}</ng-container>
   </clr-dg-column>
 
-  <clr-dg-row *customClrVirtualRows="let vm of data.vms" [clrDgItem]="vm">
-    <clr-dg-cell *ngFor="let column of columns">{{ vm[column.field] }}</clr-dg-cell>
-  </clr-dg-row>
+  <ng-template customClrVirtualRows let-vm [customClrVirtualRowsOf]="data.vms" [customClrKeyboardScrollPageSize]="100">
+    <clr-dg-row [clrDgItem]="vm">
```

Comment:
You don't have to de-sugar this. `<clr-dg-row *customClrVirtualRows="let vm of data.vms; keyboardScrollPageSize: 100" [clrDgItem]="vm">...</clr-dg-row>`

Reply:
I didn't know it worked that way. Fixed

Reply:
All good!